queries/python/textobjects.scm

```diff
@@ -34,28 +34,182 @@
 (call (_) @call.inner)
 
 ;; Parameters
-((parameters
-  ([(identifier)
-   (tuple)
-   (typed_parameter)
-   (default_parameter)
-   (typed_default_parameter)
-   (list_splat)
-   (dictionary_splat)] @parameter.inner)
-   . ","? @_end)
-  (#make-range! "parameter.outer" @parameter.inner @_end))
-
-((lambda_parameters
-  [(identifier)
-   (tuple)
-   (typed_parameter)
-   (default_parameter)
-   (typed_default_parameter)
-   (list_splat)
-   (dictionary_splat)] @parameter.inner
-   . ","? @_end)
-  (#make-range! "parameter.outer" @parameter.inner @_end))
+
+(
+  (
+  parameters
```

Comment:
nit: these should be aligned like `((foo`. Is in my list to create an autoformatter for query files

Reply:
fixed